Common/tests/Submitter/ExceptionInterceptorTests.cs

```diff
@@ -263,7 +277,7 @@ await foreach (var req in requests.WithCancellation(cancellationToken)
                                                                      SessionId = "SessionId",
                                                                      TaskOptions = new TaskOptions
                                                                                    {
-                                                                                     MaxDuration = Duration.FromTimeSpan(TimeSpan.MaxValue),
+                                                                                     MaxDuration = Duration.FromTimeSpan(TimeSpan.FromSeconds(1)),
```

Comment:
If you want an Infinite timespan, you could use `Timeout.Infinite`

Reply:
I just need a valid timespan for grpc. 1 second is okay.